test/ClientTest.php

```diff
@@ -524,4 +524,59 @@ public function testFormUrlEncodeSeparator()
         $rawRequest = $client->getLastRawRequest();
         $this->assertContains('foo=bar&baz=foo', $rawRequest);
     }
+
+    public function uriDataProvider()
+    {
+        return [
+            ['/example', true],
+            ['http://localhost/example', false]
+        ];
+    }
+
+    /**
+     * @dataProvider uriDataProvider
+     */
+    public function testValidRelativeURI($uri, $isValidRelativeURI)
```

Comment:
What _behavior_ is this testing, exactly? This should likely be "testUriCorrectlyDeterminesWhetherOrNotItIsAValidRelativeUri".